lib/agent0/agent0/hyperdrive/agents/hyperdrive_account.py

```diff
@@ -92,7 +94,8 @@ def liquidation_trades(self) -> list[Trade[MarketAction]]:
                             action_type=MarketActionType.CLOSE_LONG,
                             trade_amount=long.balance,
                             wallet=self.wallet,  # type: ignore
-                            maturity_time=maturity_time,
+                            # TODO this should be in int
+                            maturity_time=FixedPoint(maturity_time),
```

Comment:
why can't it be an int now?

Reply:
HyperdriveMarketAction is in elfpy, and we no touchy elfpy